packages/tasit-action/src/TasitAction.js

```diff
@@ -172,6 +204,7 @@ class TransactionSubscription extends Subscription {
 
         if (receipt !== null) {
           this.#txConfirmed = true;
```

Comment:
Is there any code path where we want to set `this.#txConfirmed` back to `false`?

Reply:
I've changed it to `this.#txConfirmations` ([uncle block PR](https://github.com/tasitlabs/TasitSDK/pull/123)) the `0` is the new `false`.
It's starts w/ 0 and can turnback to 0 when reorg occurs.

Reply:
Ah right. Okay, good.

Reply:
Resolving this